api/sbom.proto

```diff
@@ -61,9 +61,10 @@ message Metadata {
     string version = 2; // Int in CDX, but lets string it to capture other possible schemes
     string name = 3;
     google.protobuf.Timestamp date = 4; // created date in spdx
-    repeated Tool tools = 5;
+    repeated Tool tools = 5; // TODO: note SPDX 3.0 and CDX 1.5 migrated to use Element (or subclass of Element) so we should too, use Node instead.
     repeated Person authors = 6;
     string comment = 7;
+    repeated Lifecycle lifecycles = 8;
```

Comment:
Rename this to `Type` to reflect SBOM type as coined by the SBOM working groups (see other comment).
```suggestion
    repeated DocumentType type = 8;
```